test_runner/regress/test_download_extensions.py

```diff
@@ -104,7 +77,7 @@ def endpoint_handler_build_tag(request: Request) -> Response:
 
     # Start a compute node with remote_extension spec
     # and check that it can download the extensions and use them to CREATE EXTENSION.
-    env = neon_env_builder.init_start()
+    env = neon_env_builder_local.init_start()
     tenant_id, _ = env.neon_cli.create_tenant()
```

Comment:
Drive-by: there is already the initial tenant and timeline created.

Reply:
One could avoid creating them with `init_config`.

Reply:
please, check latest commit
agree that extra tenant is not needed